drools-wb-extensions/drools-wb-drl-extension/drools-wb-drl-extension-server/src/main/java/org/kie/workbench/common/services/datamodel/backend/server/cache/DSLExtension.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (C) 2017 Red Hat, Inc. and/or its affiliates.
```

Comment:
I believe the ```(C)``` is surplus to requirements and not part of the _standard_ licence.